data/beps_completed.yml

```diff
@@ -19,20 +21,22 @@
     pull_request_merged: 2021-02
 
 -   number: '002'
-    title: BIDS Models Specification
+    title: BIDS Statistical Models Specification
+    link: https://bids-standard.github.io/stats-models/
     leads:
     -   name: Tal Yarkoni
     -   name: Chris Markiewcz
     content:
     -   file format
-    outcome: Became an [independent specification with validator](https://bids-standard.github.io/stats-models/) to execute statistical models, with minimal
+    outcome: Became an [independent specification with validator](https://bids-standard.github.io/stats-models/){:target="_blank"} to execute statistical models, with minimal
```

Comment:
This table doesn't have an outcome column, and this appears to be an outlier for that case.